packages/utils-compat-data/package.json

```diff
@@ -0,0 +1,94 @@
+{
+    "ava": {
+        "failFast": false,
+        "files": [
+            "dist/tests/**/*.js"
+        ],
+        "timeout": "1m"
+    },
+    "dependencies": {
+        "@types/parse5": "^5.0.2",
```

Comment:
Is this ok as `dependency` or should it be a devDependency?

Reply:
we might not even need a lot of this stuff. I think I copy pasted from the original utils and didn't do a lot of clean up.
Looking forward to #3104 to get merged

Reply:
I've removed all the unused dependencies